mitodl/odl-video-service
# Sort collections by reverse creation time

#### What are the relevant tickets?
Fixes #211 

#### What's this PR do?
Adds a `created_at` field to the Collection model and sets its default sort order to created_at descending.

#### How should this be manually tested?
- `docker-compose run web python manage.py migrate`
- create a few new collections
- Your collection list should show the new collections in the reverse order that you created them.
